cmd/root.go

```diff
@@ -24,13 +10,75 @@ import (
 	"text/template"
 
 	"github.com/gobuffalo/packr"
-	"github.com/okkur/reposeed/cmd/reposeed/config"
-	templatesBox "github.com/okkur/reposeed/cmd/reposeed/templates"
+	"github.com/okkur/reposeed/cmd/config"
+	templatesBox "github.com/okkur/reposeed/cmd/templates"
+	"github.com/spf13/cobra"
 	yaml "gopkg.in/yaml.v2"
 )
 
-const SupportedConfigVersion = "v1"
+var outputDir, conf string
+var overwrite bool
+
+var rootCmd = &cobra.Command{
+	Use:   "reposeed",
+	Short: "Extensive repository base files up and running in an instant",
+	Long: `
+	Start with the base layer necessary to focus on your project and not on the repository.  
+	Licensing, structure, documentation and more boilerplate to get you started from your first commit.`,
+	Example: "reposeed -c .seed-config.yaml -o ~/myproject",
+	Run:     RootHandler,
+}
+
+func init() {
+	rootCmd.PersistentFlags().StringVarP(&outputDir, "output", "o", "", "Output directory")
+	rootCmd.PersistentFlags().StringVarP(&conf, "conf", "c", ".seed-config.yaml", "Config file")
+	rootCmd.PersistentFlags().BoolVarP(&overwrite, "overwrite", "w", false, "Force overwrite files")
+}
+
+func RootHandler(cmd *cobra.Command, args []string) {
+	box := templatesBox.GetTemplates()
+
+	config := parseConfig(conf)
+	bl := make(map[string]bool)
+	bl["seed-config.example.yaml"] = true
+	configVersion := config.Reposeed.ConfigVersion
+	templates := parseTemplates(box)
 
+	if configVersion == SupportedConfigVersion {
+		for _, templateName := range box.List() {
+			file, _ := box.Open(templateName)
+			fileStat, _ := file.Stat()
+			if bl[fileStat.Name()] {
+				log.Println(filepath.SkipDir)
```

Comment:
Does this print the skipped directory?

Reply:
Used the filename instead of `filepath.SkipDir` in the last commit.